src/DynamoCore/Configuration/PathManager.cs

```diff
@@ -50,12 +50,35 @@ struct PathManagerParams
 
     internal class PathManager : IPathManager
     {
-        internal static Lazy<PathManager>
-         lazy =
-         new Lazy<PathManager>
-         (() => new PathManager(new PathManagerParams()));
+        private static Lazy<PathManager> lazy;
```

Comment:
Why is this implementation `Lazy` the docs say:
```
Use lazy initialization to defer the creation of a large or resource-intensive object, or the execution of a resource-intensive task, particularly when such creation or execution might not occur during the lifetime of the program.
```

I think it's unlikely any of our clients will not need a path manager.

If it was to make it thread safe, well then, this PR breaks that invariant, so whats the point?